release.nix

```diff
@@ -178,41 +178,23 @@ let
             preBuild = (oldAttributes.preBuild or "") +
               pkgs.lib.optionalString pkgs.stdenv.isDarwin ''
                 export DYLD_LIBRARY_PATH=${grpc}/lib''${DYLD_LIBRARY_PATH:+:}$DYLD_LIBRARY_PATH
+              '' +
+              pkgs.lib.optionalString pkgs.stdenv.isLinux ''
+                export LD_LIBRARY_PATH=${grpc}/lib''${LD_LIBRARY_PATH:+:}$LD_LIBRARY_PATH
```

Comment:
I'm a little surprised you needed to add this for linux since we haven't needed it in the past. I also think there must exist a better idiom for propagating the library path to the build for this dependency but we don't need to solve that in this PR.

Reply:
`shellFor` works nicely for getting a multi-package shell which also has `grpc` available but the way the test-suite is setup you cannot just run it in `new-test` in the resulting shell since it assumes that `grpc-haskell-core` is in the global package db. We might want to refactor this at some point.

Reply:
I see, thank you for explaining the latter bit about `new-test`.